packages/core/src/tracing/index.js

```diff
@@ -133,8 +144,8 @@ exports.isLatestEsmSupportedVersion = isLatestEsmSupportedVersion;
  * @param {string} instrumentationKey
  */
 const isInstrumentationDisabled = (cfg, instrumentationKey) => {
-  const extractedInstrumentationName = instrumentationKey.match(/.\/instrumentation\/[^/]*\/(.*)/)[1];
-
+  const matchResult = instrumentationKey.match(/.\/instrumentation\/[^/]*\/(.*)/);
```

Comment:
Q: Could you pls add a comment why we need the logic? Its a bit hard to understand later.